Assets/Scripts/GameCalculator.cs

```diff
@@ -4,23 +4,27 @@
 
 public class GameCalculator : MonoBehaviour
 {
-    [SerializeField] private List<FieldsClickController> _field;
+    [SerializeField] private List<FieldClickController> _fields;
 
     [SerializeField] public List<FieldValue> _fieldValues;
```

Comment:
no public serialized fields. If you want to access _fieldValues outside of this script just create a public get property for this field